docs/source/protocol.rst

```diff
@@ -103,12 +89,17 @@ Event ID
 - Listed in :ref:`events page`
 - See: http://linux.die.net/man/3/uuid
 
-Relative Timestamp
-++++++++++++++++++
+OS Version
+++++++++++
+
+- String
+- GVariant symbol: ``s``
```

Comment:
Could document that this is the `VERSION_ID` from [os-release](https://www.freedesktop.org/software/systemd/man/os-release.html).

Reply:
Can we document this?